lib/ansible/modules/windows/win_file.ps1

```diff
@@ -25,6 +25,7 @@ $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -default
 
 $path = Get-AnsibleParam -obj $params -name "path" -type "path" -failifempty $true -aliases "dest","name"
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -validateset "absent","directory","file","touch"
+$attributes = Get-AnsibleParam -obj $params -name "options" -type "str" -aliases "attributes"
```

Comment:
I really prefer seeing the same parameters as the **file** module, and in this case make **attributes** the parameter, not **options**. (Not sure why this is called options here)

https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/files.py#L75